src/api/spec/features/webui/comments_spec.rb

```diff
@@ -13,13 +14,10 @@
   end
 
   scenario 'can be answered' do
-    skip_if_bootstrap
```

Comment:
@eduardoj I think you haven't understood @dmarcoux proposal, if you remove this `skip` then you need to drop `src/api/spec/bootstrap/features/webui/projects/comments_spec.rb`